lib/truffle/java/java.rb

```diff
@@ -122,6 +127,10 @@ def inspect
       super.to_s
     end
 
+    def const_get(name, inherit=true)
+      super.const_get(name, inherit) rescue method_missing(name)
```

Comment:
I think I'd assumed it was needed to get round invalid const names being used for Java packages, but they're only checked for on const_set. I'll delete it.